.github/components.yml

```diff
@@ -149,6 +149,7 @@ PyTorch_FE:
   build:
     - CPU
     - Python_API
+    - TOKENIZERS # PyTorch_FE tests depend on tokenizers build
```

Comment:
please clarify - what in PT FE depends on tokenizers?

Reply:
looks like you have referred to TF tests, because PT ones don't depends on Tokenizers:
https://github.com/openvinotoolkit/openvino/blob/d380763d8f2e8d99cca17606542ded2245215286/.github/workflows/ubuntu_22.yml#L313-L321

Reply:
I referred to PyTorch Model Tests
https://github.com/openvinotoolkit/openvino/blob/d380763d8f2e8d99cca17606542ded2245215286/.github/workflows/ubuntu_22.yml#L369C1-L376C31

Reply:
@rkazants then it's you who added tokenizers dependency for PT tests https://github.com/openvinotoolkit/openvino/commit/c7b4ba34c345f25d58d98cf7bae86e8492bb0fec